mark-nonconverted-reads.py

```diff
@@ -92,23 +92,29 @@ def parse_bam(bam_file, fasta_dict, out, args):
         nonconverted_dict[chrom] = 0
         
     for read in bam_file:
-    
-        # Ignore reads not part of a proper pair
+
+        if read.reference_id == -1:
```

Comment:
why is this needed?

Reply:
I struggled with a bam with unmapped reads and pysam's error messages were not very helpful... I can take it out if we should NEVER encounter these reads?